dataikuapi/dss/recipe.py

```diff
@@ -1397,6 +1398,75 @@ def with_output_evaluation_store(self, mes_id):
         return self._with_output(mes_id, role="evaluationStore")
 
 
+class StandaloneEvaluationRecipeCreator(DSSRecipeCreator):
+    """
+    Builder for the creation of a new "Standalone Evaluate" recipe, from an
+    input dataset
+
+    .. code-block:: python
+
+        # Create a new standalone evaluation of a scored dataset
+
+        project = client.get_project("MYPROJECT")
+        builder = StandaloneEvaluationRecipeCreator("my_standalone_evaluation_recipe", project)
+        builder.with_input("scored_dataset_to_evaluate")
+        builder.with_output_evaluation_store(evaluation_store_id)
+
+        new_recipe = builder.build()
```

Comment:
I can't find why, but this creates an invalid recipe. If you just do this step and go check the recipe in the GUI, you will have a JS error.
Either we find a way to properly initialise it, or we clearly state it in the documentation.
I would prefer the first option, but I couldn't figure how to do it.

Reply:
I am very sorry, but I just tested and I got no error in the GUI. Can you please provide additional details?

Reply:
I've just created a SER and stopped after `builder.build()`. I do not update the settings.

The recipe is created and appears in the flow, but when you open it, it's empty and I have a JS error:
![image](https://user-images.githubusercontent.com/10883487/148039057-776fa404-8368-4bf6-a7dc-43324620690a.png)

Reply:
I managed to reproduce. I am working on it.

Reply:
@michaelbourhis I pushed a fix on the backend side of DSS, not on the public api. 

It seemed the backend was trying to force the casting of the output to a dataset-type output (only for the publicapi though, everything was working fine when you worked your way through the GUI to create a SER, as you already know.)

I can only invite you to test it again using the `feature/dss100-sc-74814-add-the-ability-to-create-a-standalone-evaluation` branch for both your DSS and your API client.

See: https://github.com/dataiku/dip/pull/14421

Reply:
That's ok now with that fix, thank you!